src/fstapp_server.erl

```diff
@@ -6,9 +6,15 @@
 -export([init/1, handle_call/3, handle_cast/2, handle_info/2,
 	terminate/2]).
 
+-type field_data() :: {term()}.
```

Comment:
this is wrong, you are expecting something like `{cpu, Number}` or `{disk, Number}`